src/main/java/com/bartek/Medical/Model/DoctorDTO.java

```diff
@@ -0,0 +1,10 @@
+package com.bartek.Medical.Model;
+
+import lombok.Data;
+
+@Data
+public class DoctorDTO {
```

Comment:
Dlaczego nie ma email? I dlaczego nie ma id?

Reply:
Wydaje mi się że użytkownikowi nie jest potrzebny Id jednak z emailem to służna uwaga

Reply:
Dlaczego id nie jest potrzebny a co gdy uzytkownik dopiero zaklada swoje konto bądź w sytuacji gdy mamy aplikację RegistrationService jednym z jej kroków jest zlecenie utworzenia nowego doktora aczkolwiek nie jest to tylko jedyny krok, następnie np przypisywany jest on jeszcze do odpowiedniego szpitala w takiej sytuacji skąd serwis orkiestrujący proces rejestracji miałby wiedzieć jaki id ma doctor, ktory został utworzony?